contracts/VestingSwap.sol

```diff
@@ -31,6 +45,9 @@ contract VestingSwap is Ownable {
     mapping(address => SwapInfo) public swap;
     mapping(address => uint) public swappedAmountOf;
 
+    event AlphaSwapInitialized();
```

Comment:
Events are designed for client application. Empty events are not needed. Pass `start` and `percentages`.